patches/server/0739-Add-environment-variable-to-set-velocity-secret.patch

```diff
@@ -0,0 +1,19 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Noah van der Aa <ndvdaa@gmail.com>
+Date: Thu, 5 Aug 2021 16:08:23 +0200
+Subject: [PATCH] Add environment variable to set velocity secret.
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperConfig.java b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+index 95d8cf449bf0439fddaf319246fef51e1570b68e..8e497a05e3d430cd2047788b032bca6fcb6c7bda 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+@@ -356,7 +356,7 @@ public class PaperConfig {
+     private static void velocitySupport() {
+         velocitySupport = getBoolean("settings.velocity-support.enabled", false);
+         velocityOnlineMode = getBoolean("settings.velocity-support.online-mode", false);
+-        String secret = getString("settings.velocity-support.secret", "");
++        String secret = java.util.Objects.requireNonNullElse(System.getenv("VELOCITY_SECRET"), getString("settings.velocity-support.secret", ""));
```

Comment:
Could this be not fully-qualified?

Reply:
It would require an extra import, as far as I know Paper uses FQNs for what would've otherwise been a "single use" import.

Reply:
Ah, very nice to know! Wasn't sure of the conventions, thanks for sharing 👍